src/eradiate/scenes/atmosphere/_particle_layer.py

```diff
@@ -219,6 +219,19 @@ def _bottom_top_validator(self, attribute, value):
         default="False",
     )
 
+    particle_shape: str = documented(
+        attrs.field(
+            default="spherical",
+            kw_only=True,
+        ),
+        doc="Defines the shape of the particle. Only used in polarized mode."
+        "* spherical: 4 coefficients considered [m11, m12, m33, m34]."
+        "* spheroidal: 6 coefficients considered [m11, m12, m22, m33, m34, m44].",
```

Comment:
```suggestion
        doc="Defines the shape of the particle. Only used in polarized mode.\n\n"
        '* ``"spherical"``: 4 coefficients considered [m11, m12, m33, m34].\n'
        '* ``"spheroidal"``: 6 coefficients considered [m11, m12, m22, m33, m34, m44].',
```
See [PR docs build for a render](https://eradiate--445.org.readthedocs.build/en/445/rst/reference_api/generated/autosummary/eradiate.scenes.phase.TabulatedPhaseFunction.html#eradiate.scenes.phase.TabulatedPhaseFunction).